### PR TITLE
Add-Breadcrumb-to-Sign-up-Page

### LINKED ISSUE #27: Add a breadcrumb to the Sign Up page
A breadcrumb is used for  navigation convenience.

Usually you see it at the top of the main content section, reading like:

`Home / Sign Up`

It shows you the steps the user took to reach the page. Usually, from home, it goes to Sign up.

The word home should be linked to the home/index page.

Reference: https://developer.mozilla.org/en-US/docs/Glossary/Breadcrumb

### PR DESCRIPTION
**Pull request by karlasanc**
Resolve #27: I made modifications on two files (signup.html & signup.css). 

- Within the signup.html document, I enclosed the breadcrumb unordered list within navigation HTML tags to add semantic meaning, used the aria-label attribute to ensure it is compatible with accessibility devices, and added a breadcrumb class to the navigation tags to style it with CSS. The home list item includes an anchor tag with a href attribute that links the index.html page for users to go back. 

- Within the signup.css document, I used the class selector to change the unordered list from vertical to horizontal using the display attribute, removed the bullet points using the list-style attribute, and changed the color to separate the breadcrumb nav from the sign-up form. I targeted the space between list items to include padding and forward slashing using "/\00a0". 

image

### CODE REVIEW

**Comment by nbkhope** [comment]
Please add to the beginning of description the issue number, like this:

```
Resolve #123
```

File: src/signup.css [hunk 1]
```diff
@@ -41,4 +41,24 @@ form input[type='submit'] {
...
+/*BreadCrumb CSS*/
+.breadcrumb > ul {
+  display: flex;
+  padding: 15px;
+  list-style: none;
+  background-color: #eee;
+}
+
+/* Add a slash symbol (/) before/behind each list item */
+.breadcrumb li+li:before {
```
Comment (nbkhope) [inline src/signup.css:55]: You didnt have to be this fancy, but okay. 😆

File: src/signup.html [hunk 2]
```diff
@@ -8,6 +8,15 @@
     <link rel="stylesheet" href="signup.css">
   </head>
   <body>
+
+    <!--breadcrumbs-->
```
Comment (nbkhope) [inline src/signup.html:12]: No need for comments like this when it's obvious from reading the code.

Usually comments are to add further context not apparent from reading the code.

```suggestion
```

**Comment by karlasanc** [comment]
Updated description, thanks!

**Review by nbkhope — APPROVED** [review]

**Comment by nbkhope** [comment]
@karlasanc looks great, thank you

**Pull request merged**